nixos/modules/services/misc/libreddit.nix

```diff
@@ -0,0 +1,66 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+  let
+    cfg = config.services.libreddit;
+
+    args = concatStringsSep " " ([
+      "--port ${toString cfg.port}"
+      "--address ${cfg.address}"
+    ] ++ optional cfg.redirect "--redirect-https");
```

Comment:
This implies we need an ssl certificate. I don't see one. Can you please explain?

Reply:
This feature was removed.

Reply:
Which feature? The SSL redirect feature still seems to be in your latest PR version - were you referring to a different feature?